docs/Data_correction_documentary_cleared.ipynb

```diff
@@ -0,0 +1,290 @@
+  {
+   "cell_type": "markdown",
+   "id": "788a4fce-63dc-4c23-8d2a-e9a05db0fdf7",
+   "metadata": {},
+   "source": [
+    "## Data correction procedure for TOF-SANS with polarization analysis\n",
+    "\n",
+    "- Note on notation:\n",
+    "> - subscripts = discretization variable from measurement\n",
+    "> - superscripts = Names\n",
+    "> - () = continuous function of....\n",
+    "\n",
+    "| Value | Meaning | shape | How acquired? | \n",
+    "|:--------:|:--------:|:--------:|:--------:|\n",
+    "|  $I^{++}_{t,\\lambda}$   |  uncorrected sample data of ++ channel |  data array with columns for $I$, $\\lambda$, $t$   |  neutron data |\n",
+    "|  $I^{+-}_{t,\\lambda}$   |  uncorrected sample data of +- channel |  data array with columns for $I$, $\\lambda$, $t$   | neutron data    |\n",
+    "|  $I^{-+}_{t,\\lambda}$   |  uncorrected sample data of -+ channel  |  data array with columns for $I$, $\\lambda$, $t$   | neutron data   |\n",
+    "|  $I^{--}_{t,\\lambda}$   |  uncorrected sample data of -- channel |  data array with columns for $I$, $\\lambda$, $t$    | neutron data   |\n",
+    "|  $S^{++}_{t,\\lambda}$   |  corrected sample data of ++ channel  |  data array with columns for $I$, $\\lambda$, $t$   | by pol-correction  |\n",
+    "|  $S^{+-}_{t,\\lambda}$   |  corrected sample data of +- channel  |  data array with columns for $I$, $\\lambda$, $t$   | by pol-correction   |\n",
+    "|  $S^{-+}_{t,\\lambda}$   |  corrected sample data of -+ channel  |  data array with columns for $I$, $\\lambda$, $t$   | by pol-correction   |\n",
+    "|  $S^{--}_{t,\\lambda}$   |  corrected sample data of -- channel  |  data array with columns for $I$, $\\lambda$, $t$   | by pol-correction    |\n",
+    "|  $p^{cell}$   |  pressure of <sup>3</sup>He-polarizer/analyzer  |  scalar, 1-dim   | from metadata  |\n",
+    "|  $l^{cell}$   |  length of <sup>3</sup>He-polarizer/analyzer   |  scalar, 1-dim   | from metadata   |\n",
+    "|  $T^{g, cell}(\\lambda)$   |  transmission of empty glass of polarizer/analyzer   |  scalar, 1-dim   | from metadata  |\n",
+    "|  $O^{cell}(\\lambda)=O^{0, cell}\\cdot\\lambda$   |  wavelength-dependent opacity of <sup>3</sup>He-polarizer/analyzer  |  continuous function of $\\lambda$   | $O^{0, cell}$ fit to data |\n",
+    "|  $O^{0, cell}$   |  wavelength-independent pre-factor of opacity of <sup>3</sup>He-polarizer/analyzer   |  scalar, 1-dim   | fit to data  |\n",
+    "|  $P^{^3He, cell}(t) = C^{cell}e^{-t/T^{1, cell}}$   |  time-dependent nuclear polarization of <sup>3</sup>He-polarizer/analyzer  |  continuous function of $t$   | $C^{cell}, T^{1, cell}$ fit to data   |\n",
+    "|  $C^{cell}$   |  Pre-factor of nuclear polarization exponential decay of <sup>3</sup>He-polarizer/analyzer  |  scalar, 1-dim   | fit to data | $P_{He0}$   |\n",
+    "|  $T^{1, cell}$   |  Decay constant of nuclear polarization exponential decay of <sup>3</sup>He-polarizer/analyzer  |  scalar, 1-dim   | fit to data |\n",
```

Comment:
This appears to be a time-scale, can we use $t$ instead of $T$? As it is it is also easily confused with the other uses of $T$ for transmission.

Reply:
Hm. Its a commonly used "T", as its not varying in time, its a fix constant. I feel it can be dangerous to exchange with 't' as well. I see the point of confusion with Transmission. Sometims these time-decays are also called "Gamma" (large greek character). I have no objection calling it "Gamma_1". But I believe I had called it "Gamma" before Hal said that for 3He cell neutron polarization its commonly "T_1". That may be a group decision here?

Reply:
Hal recommends changing the Transmission from "T" to "Tr" instead, and leaving "T_1" as it is. Fine with me